data/mods/innawood/scenario_whitelist.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "type": "SCENARIO_BLACKLIST",
+    "subtype": "whitelist",
+    "scenarios": [
+      "evacuee",
+      "medieval",
+      "migo_prisoner",
```

Comment:
NPCs can still spawn with "futuristic" stuff I think

Reply:
They can. As can the player character. The short lore explanation I have been working with is that the player character fell through a portal to an alternative earth where humans never existed. Any other humans would likewise come from another reality, probably the same one. So it would make sense for them to have modern gear.
However, maybe this is bad, as it could incentivise the player to go out and hunt other humans for rare items? I'll add a note for it under "Describe alternatives you've considered".